lib/dovico/model/task_group.rb

```diff
@@ -0,0 +1,4 @@
+module Dovico
+  class TaskGroup < Assignment
```

Comment:
May it make sense to group various assignment types in `Dovico::Assignment::` ?
Like `Dovico::Assignment::TaskGroup`, `Dovico::Assignment::Task` ?

```
module Dovico
  class Client < Assignment
```
is a bit confusing, I'm thinking rather of an API client then of an assignment